chrome/manifest.development.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.8.2",
+  "version": "1.9.0",
```

Comment:
If this is intention to change version in PR then OK

Reply:
yes I need a new version to trigger the migration logic